docs/installing-open-mpi/configure-cli-options/installation.rst

```diff
@@ -32,48 +32,112 @@ be used with ``configure``:
   Be sure to read the description of ``--without-memory-manager``,
   below; it may have some effect on ``--enable-static``.
 
-* ``--disable-wrapper-rpath``:
-  By default, the wrapper compilers (e.g., ``mpicc``) will enable
-  "rpath" support in generated executables on systems that support it.
-  That is, they will include a file reference to the location of Open
-  MPI's libraries in the application executable itself.  This means
-  that the user does not have to set ``LD_LIBRARY_PATH`` to find Open
-  MPI's libraries (e.g., if they are installed in a location that the
-  run-time linker does not search by default).
-
-  On systems that utilize the GNU ``ld`` linker, recent enough versions
-  will actually utilize "runpath" functionality, not "rpath".  There
-  is an important difference between the two:
-
-  #. "rpath": the location of the Open MPI libraries is hard-coded into
-     the MPI/OpenSHMEM application and cannot be overridden at
-     run-time.
-  #. "runpath": the location of the Open MPI libraries is hard-coded into
-     the MPI/OpenSHMEM application, but can be overridden at run-time
-     by setting the ``LD_LIBRARY_PATH`` environment variable.
+* ``--disable-wrapper-runpath`` / ``--disable-wrapper-rpath``: By
+  default, the wrapper compilers (e.g., ``mpicc``) will enable
+  "runpath" or "rpath" support in generated executables on systems
+  that support it.  That is, they will include a filesystem path
+  reference to the location of Open MPI's libraries in the application
+  executable itself.  This means that the user does not have to set
+  ``LD_LIBRARY_PATH`` to find Open MPI's libraries, which can be
+  helpful if they are installed in a location that the run-time linker
+  does not search by default.
+
+  .. note:: By default, the wrapper compilers prefer "runpath"
+            behavior over "rpath" behavior.
+
+            * Using ``--disable-wrapper-runpath`` alters this
+              preference: "runpath" behavior will never be used.
+              Instead, the wrapper compilers will use "rpath" behavior
+              if it is supported.
+            * Using ``--disable-wrapper-rpath`` will disable *both*
+              "runpath" and "rpath" behavior in the wrapper compilers.
+
+  .. important:: Note that the ``--disable-wrapper-runpath`` and
+                 ``--disable-wrapper-rpath`` CLI options are *only*
+                 relevant to the wrapper compilers.  They do not
```

Comment:
> "are only relevant to the flags the wrapper compilers add when compiling user applications."

The wrapper compilers themselves are still built with whatever flags OMPI was built with.